pkg/scan/process_image.go

```diff
@@ -30,6 +31,25 @@ import (
 
 var (
 	ErrmaxMalwaresExceeded = errors.New("number of secrets exceeded max-secrets")
+	execMimeTypes          = []string{"text/x-shellscript", "application/x-executable", "application/x-mach-binary", "application/x-msdownload", "application/exe", "application/x-msdos-program", "application/x-elf", "application/x-sharedlib", "application/x-pie-executable", "application/java-archive", "application/x-java-archive", "text/x-python", "application/x-batch"}
+	execExtensions         = []string{
```

Comment:
Since we use the magic number, we can drop the exec extensions checks entirely, they are not relevant in unix systems